timemachine/ff/handlers/nonbonded.py

```diff
@@ -1,27 +1,44 @@
 import base64
+import os
 import pickle
+import subprocess
+import tempfile
 import warnings
-from collections import Counter
+from collections import Counter, defaultdict
+from functools import partial
+from shutil import which
 
 import jax.numpy as jnp
 import networkx as nx
 import numpy as np
+from Auto3D.ASE.geometry import opt_geometry
 from jax import jit, vmap
 from numpy.typing import NDArray
+from openff.recharge.aromaticity import AromaticityModel
```

Comment:
is this overwritten by `from timemachine.ff.handlers.bcc_aromaticity import AromaticityModel` down below?